### DISCUSSION TITLE
Glazed Terracotta Preview

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#30] [Screenshots]
![Minecraft](https://user-images.githubusercontent.com/1903667/116769975-c44ef780-aa05-11eb-8a20-df9a38765aba.png)
![Minecraft](https://user-images.githubusercontent.com/1903667/116769976-c44ef780-aa05-11eb-8c72-9e3db1d91eaa.png)
![Minecraft](https://user-images.githubusercontent.com/1903667/116769978-c4e78e00-aa05-11eb-9d5b-425c4b4cee4f.png)

**Comment by erich666** [comment]
Wow! The roughness maps really add to these.
